packages/webui/src/client/lib/rundownTiming.ts

```diff
@@ -746,10 +757,10 @@ export interface RundownTimingContext {
 	 * if the Part does not have an expected duration.
 	 */
 	partExpectedDurations?: Record<string, number>
-	/** Budget durations of segments (sum of parts budget durations). */
-	segmentBudgetDurations?: Record<string, number>
 	/** Remaining time on current part */
 	remainingTimeOnCurrentPart?: number
+	/** Remaining budget on current segment, if its countdownType === CountdownType.SEGMENT_BUDGET_DURATION */
+	remainingBudgetOnCurrentSegment?: number
```

Comment:
```suggestion
	/** Remaining budget on current segment, if its countdownType === CountdownType.SEGMENT_BUDGET_DURATION, `undefined` otherwise */
	remainingBudgetOnCurrentSegment?: number | undefined
```